quiche/src/recovery/bbr/mod.rs

```diff
@@ -252,17 +252,19 @@ impl State {
 }
 
 // When entering the recovery episode.
-fn bbr_enter_recovery(r: &mut Recovery, now: Instant) {
+fn bbr_enter_recovery(r: &mut Recovery) {
     r.bbr_state.prior_cwnd = per_ack::bbr_save_cwnd(r);
 
-    r.congestion_window = r.bytes_in_flight.max(r.max_datagram_size);
-    r.congestion_recovery_start_time = Some(now);
+    // Temporarily add lost bytes, because bytes_in_flight
+    // doesn't have lost bytes at this point. cwnd will be
+    // calculated again in bbr_modulate_cwnd_for_recovery().
+    r.congestion_window = r.bytes_in_flight +
+        r.bbr_state.newly_acked_bytes.max(r.max_datagram_size) +
```

Comment:
The current code is doing the same thing. Since, we don't have the acked_bytes  at this point, cwnd is set correctly in bbr_modulate_cwnd_for_recovery() function. At any case bbr_modulate_cwnd_for_recovery is called after entering the recovery.